src/Account/fetchAddressInfo.js

```diff
@@ -53,29 +53,33 @@ async function fetchAddressInfo(addressObj, fetchUtxo = true) {
       }
     }
     if (fetchUtxo) {
-      const fetchedUtxo = (await self.transport.getUTXO(address));
-      const utxos = [];
-      if (balanceSat > 0) {
-        fetchedUtxo.forEach((utxo) => {
-          utxos.push({
-            satoshis: utxo.satoshis,
-            txid: utxo.txid,
-            address: utxo.address,
-            outputIndex: utxo.vout,
-            scriptPubKey: utxo.scriptPubKey,
-            // scriptSig: utxo.scriptSig,
+      const fetchedUtxoReq = (await self.transport.getUTXO(address));
```

Comment:
Why do use parens here?

Reply:
There are not. 
(It was used for a function that display and return the input `util.cl(await [...])`, which is not in the pushed code (obviously), updating as per request)

Reply:
Resolved. (thx so much @shumkov for your time on this review !)